lib/winston-papertrail.js

```
@@ -136,6 +136,40 @@ var Papertrail = exports.Papertrail = function (options) {
     // Open the connection
     connectStream();
 
+    function cleanup() {
+        self._erroring = true;
+        try {
+            if (self.socket) {
+                self.socket.destroy();
+
+                //this throws some uncatchable thing, eww.
+                //self.socket.end();
+                self.socket = null;
+            }
+        }
+        catch (e) { }
```

So we're just catching here and swallowing the error. I'm wondering if we should emit it?